src/dodal/devices/eiger.py

```diff
@@ -286,6 +293,9 @@ def set_num_triggers_and_captures(self) -> Status:
 
     def _wait_for_odin_status(self) -> Status:
         self.forward_bit_depth_to_filewriter()
+        sleep(
+            2
+        )  # FIXME - VMXm hack - on VMXm ODIN will *occasionally* fail to start without this sleep. Need a better solution.
```

Comment:
We might be able to do some kind of cleverer wait for ODIN to be ready to start capturing. Needs careful testing on VMXm if we adjust this logic as it was only failing to start ~10% of gridscans before this change was added.

Reply:
Any idea what was causing it to fail when the sleep wasn't there?

Reply:
Not really. Some kind of race condition at the ODIN level, I think. 2 seconds is a number plucked from thin air and we might be able to lower it or put in some better logic, but it'll need testing on VMXm if we do that. Should be possible on a Tuesday or in a shutdown if you ask the scientists - we shouldn't need beam to test this.

For this issue, I'd maybe suggest doing some variant of:
```
if beamline == "vmxm":
    sleep(2)
```

and write an issue to investigate and fix this properly.